stablehlo/reference/Ops.h

```diff
@@ -73,10 +73,13 @@ Tensor evalXorOp(const Tensor &lhs, const Tensor &rhs, Type resultType);
 /// Evaluates an mlir::Region `region` using the runtime values `args`
 /// corresponding to the arguments of the entry block of the region.
 /// Interprets the operations within the entry block and returns the runtime
-/// values for the terminator's arguments.
-/// Assumes that the region has only one block.
-llvm::SmallVector<Tensor> eval(Region &region, llvm::ArrayRef<Tensor> args,
-                               Scope *parentScope = nullptr);
+/// values for the terminator's arguments. The optional callback `fallback` is
+/// used for evaluating ops, other than `func::returnOp` and the ones in
+/// stablehlo dialect, within the `region`. Assumes that `region` has only one
```

Comment:
"for evaluating ops, other than `func::returnOp` and the ones in stablehlo dialect, within the `region`". How about just "for evaluating ops which are not supported by the interpreter"? Rationale:
  1) There's also `func::CallOp` that we'll need to support eventually.
  2) These ops can be outside the region, e.g. if we call into another function.